workalendar/asia/singapore.py

```diff
@@ -53,7 +54,16 @@ class Singapore(WesternMixin, IslamicMixin, ChineseNewYearCalendar):
         2018: date(2018, 11, 6),
         2019: date(2019, 10, 27),
         2020: date(2020, 11, 14),
-        2021: date(2021, 11, 4),  # This might change
+        2021: date(2021, 11, 4),
+        2022: date(2022, 10, 24),
+        2023: date(2023, 12, 11),
```

Comment:
Should this be `date(2023, 11, 12)`? Or even `date(2023, 11, 13)`?
The website you reference says that the public holiday is on the 13th November 2023 in lieu of the 12th.

Reply:
I agree with @jack-pace.

```suggestion
        2023: date(2023, 11, 13),
```